virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -81,6 +81,8 @@ This release adds new features and enhancements related to the following compone
 //CNV-29984: RN - Document auto memory and cpu resource limits
 * Enabling the `AutoResourceLimits` feature gate automatically manages CPU and memory limits of a VM.
 
+* The KubeVirt Tekton tasks are now shipped as a part of the link:https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks[{product-title} Pipelines catalog].
```

Comment:
@skopacz1 Yes, I am afraid it is. That Git repo is the only place where they keep this catalog. In the next release, it will move to catalog.redhat.

Reply:
Problem is, that currently we don't have official RH catalog for tekton objects. The OCP pipelines people are planning to link the pipeline object to catalog.redhat.com, but currently, the connection is not finished.

Reply:
That is unfortunate. I believe the spirit of the rule against linking to a repo is to account for external owners changing the directory structure and such, and I suppose that this repo is somewhat internal. I guess if there are plans to avoid having to like to this repo in the future, the link can stay this time.